src/BugsnagUnity/Client.cs

```diff
@@ -39,7 +39,9 @@ internal class Client : IClient
 
     bool InForeground => ForegroundStopwatch.IsRunning;
 
-    const string UnityMetadataKey = "Unity";
+    const string AppMetadataKey = "App";
+
+    const string DeviceMetadataKey = "Device";
```

Comment:
These section names should be lower case, in line with the other data (I'm not sure why "Unity" was upper case, but this should fix it going forward):

```suggestion
    const string AppMetadataKey = "app";

    const string DeviceMetadataKey = "device";
```

unfortunately, this will touch all the tests.

Reply:
done